ray-operator/test/migration/README.md

```
@@ -0,0 +1,241 @@
+```bash
+./post-migration-check.sh test-migration /tmp/migration-check/pre /tmp/migration-check/post
+```
+
+This verifies:
+- CodeFlare items were removed
+- Custom items were preserved
+- Generates pass/fail report
+
+### Example Output
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)